azurerm/internal/services/cosmos/cosmosdb_account_resource.go

```diff
@@ -182,6 +181,11 @@ func resourceArmCosmosDbAccount() *schema.Resource {
 							Required:     true,
 							ValidateFunc: validation.IntAtLeast(0),
 						},
+
+						"is_zone_redundant": {
```

Comment:
in Terraform we intentionally avoid `is_` prefixes - so we can make this `zone_redundant` to match the other resources

Reply:
Changed